apps/wallet/src/ui/app/pages/home/apps/index.tsx

```diff
@@ -1,33 +1,67 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { useFeature } from '@growthbook/growthbook-react';
 import { Route, Routes } from 'react-router-dom';
 
 import { Content } from '_app/shared/bottom-menu-layout';
 import FiltersPortal from '_components/filters-tags';
 import AppsPlayGround, { ConnectedAppsCard } from '_components/sui-apps';
+import { FEATURES } from '_src/shared/experimentation/features';
+
+import type { DAppEntry } from '_src/ui/app/components/sui-apps/SuiApp';
 
 import st from './AppsPage.module.scss';
 
+type FilterTag = {
+	name: string;
+	link: string;
+};
+
 function AppsPage() {
-	const filterTags = [
+	const defaultFilterTags: FilterTag[] = [
 		{
-			name: 'Playground',
-			link: 'apps',
+			name: 'Connections',
+			link: 'apps/connected',
 		},
 		{
-			name: 'Active Connections',
-			link: 'apps/connected',
+			name: 'All',
+			link: 'apps',
 		},
 	];
+	const ecosystemApps = useFeature<DAppEntry[]>(FEATURES.WALLET_DAPPS).value ?? [];
+
+	const uniqueAppTagNames = new Set<string>();
+
+	ecosystemApps
```

Comment:
and this are not necessary now right?

Reply:
Yeah @Nikhil-Mysten can you open a PR to clean this up please?